cfgov/jinja2/v1/browse-basic/index.html

```diff
@@ -185,6 +185,95 @@ <h2 class="header-slug">
         {% endfor %}
     {% endif %}
 
+
+    {# TODO: Remove the below when input-contains-icon
+             class is added cf-enhancements. #}
+    <div class="block">
+        <div class="block">
+            <div class="input-contains-icon">
+                <div class="input-contains-icon_before
+                            input-contains-icon_before__search">
+                    <span class="input-contains-icon_icon"></span>
```

Comment:
It's needed if the spans are wrapped in a button (which I need to push a fix for). I like the idea of removing the spans. Do you see a way to support both these scenarios with a reduced number of spans?:
```
.div
   .cf-icon
   .label
```

And:

```
div
   .btn
      .cf-icon
      .label
```